maven-java/kalix-maven-plugin/src/main/scala/kalix/RunAllMojo.scala

```diff
@@ -11,19 +10,13 @@ import org.apache.maven.plugins.annotations.ResolutionScope
  */
 @Mojo(name = "runAll", defaultPhase = LifecyclePhase.VALIDATE, requiresDependencyResolution = ResolutionScope.RUNTIME)
 @Execute(phase = LifecyclePhase.COMPILE)
-class RunAllMojo extends RunParameters with DockerParameters {
+class RunAllMojo extends RunParameters {
 
   override def execute(): Unit = {
-
-    val dockerComposeUtils = DockerComposeUtils(dockerComposeFile, sys.env)
-    dockerComposeUtils.start()
```

Comment:
When running Kalix using the plugin, we might want to start the docker-compose from inside the plugin though. So we might want to revert this change later.

The rationale is that if we want to have some hot-reload feature (like Spring DevTools), we don't want to have the docker-compose process attached to the process running `Main`. 

If we have them separate, the hot-reload will mean only reload the user function and proxy will be kept in the background. 

If we run docker-compose from behind the `Main`, then it will be shutdown and restart on each reload. We don't need to decide it now. We don't have hot-reload anyway, but something to keep in mind.